index.html

```diff
@@ -561,8 +561,8 @@ <h3>The <dfn><code>pointercancel</code> event</dfn></h3>
                 <ul>
                     <li>The user agent has determined that a pointer is unlikely to continue to produce events (for example, because of a hardware event).</li>
                     <li>After having fired the <code>pointerdown</code> event, if the pointer is subsequently used to manipulate the page viewport (e.g. panning or zooming).
-                     <div class="note">User agents can trigger panning or zooming through multiple pointer types (say "touch" and "pen"),
-                     and therefore can possibly cancel all pointers of those types for a single viewport manipulation.</div</li>
+                     <div class="note">User agents can trigger panning or zooming through multiple pointer types (say touch and pen),
```

Comment:
nitpick: i'd still change "say" to "such as" (the former sounds a tad too informal/colloquial)